source/api/tests/test_api_integration.py

```diff
@@ -381,3 +382,83 @@ def test_create_upload_delete_dataset_DIMENSION_JSON_SUB_DIRECTORY(test_configs,
 
 def test_create_upload_delete_dataset_FACT_JSON_SUB_DIRECTORY(test_configs, test_data_set_type, get_etl_data_by_job_id, generate_random_test_files_to_s3_bucket):
    test_data_set_type("FACT", "JSON", test_configs, get_etl_data_by_job_id, generate_random_test_files_to_s3_bucket, s3_key_sub_dir="integ_test_data/")
+
+def test_create_upload_delete_dataset_DIMENSION_JSON_LIVE_RAMP(test_configs, test_data_set_type, get_etl_data_by_job_id, generate_random_test_files_to_s3_bucket):
+    test_columns = [
+        {
+            "name": "user_id",
+            "description": "The customer resolved id",
+            "dataType": "STRING",
+            "nullable": True,
+            "isMainUserId": True
+        },
+        {
+            "name": "user_type",
+            "description": "The customer resolved type",
+            "dataType": "STRING",
+            "nullable": True,
+            "isMainUserIdType": True
+        },
+        {
+            "name": "first_name",
+            "description": "hashed First name",
+            "dataType": "STRING",
+            "externalUserIdType": {
+            "type": "HashedIdentifier",
+            "identifierType": "FIRST_NAME"
+            },
+            "nullable": True
+        },
+        {
+            "name": "product_name",
```

Comment:
A ramp id is not going to be a "product_name". This would be a better column description:

"name": "ramp_id",
"description": "The user's LiveRamp ID",
"dataType": "STRING",
"nullable": true,
"externalUserIdType": {
    "type": "LiveRamp"
}

Reply:
Resolved